Kingno3l/invoice
# Implement Invoice UI with BeeTablesCompose Library Integration

This PR introduces the following features and enhancements to the Invoice UI in the android.example.invoice package:

**Invoice Data Model:**

Added User and Invoice data classes to represent invoice items and invoice details respectively.

**MainActivity Setup:**

Configured MainActivity to display invoice details using Jetpack Compose.
Enabled edge-to-edge display.

**MainContent Composable:**

Created MainContent composable to structure the layout of the invoice.
Integrated multiple composables including Logo, AddressSection, InvoiceTable, ItemTable, InvoiceSummary, and DownloadInvoiceButton.

**BeeTablesCompose Integration:**

Utilized BeeTablesCompose library to create tables for displaying invoice details and item lists.
Configured table headers and styling including background colors and text styles.

**UI Components:**

- [ ] InvoiceTable and ItemTable composables for displaying invoice and item details using BeeTablesCompose.
- [ ] Logo composable to display the company logo.
- [ ] AddressSection composable to show shipping address.
- [ ] InvoiceSummary composable to summarize billing information including subtotal, tax rate, total, and balance due.
- [ ] DownloadInvoiceButton composable to provide an option to download the invoice.

Screenshot:
![Screenshot_1720278146](https://github.com/Kingno3l/invoice/assets/66368728/c5defb49-e1ab-423d-8d01-8341df506b25)
![Screenshot_1720278154](https://github.com/Kingno3l/invoice/assets/66368728/8d3ce8f0-1cae-4734-89bb-5ea5f11faed7)

**Additional Notes:**

- [ ] Added custom color Green12 (#235705) for consistent styling.
- [ ] The layout ensures responsiveness and alignment for various screen sizes.

**Library Used:**
[BeeTablesCompose](https://github.com/Breens-Mbaka/Jetpack-Compose-Tables)

**How to Test:**

1. Run the application.
2. Verify the layout and data display in the invoice UI.
3. Check the functionality of the "Download Invoice" button.
